### PR TITLE
moving asb stage cases from cucushift to verification-test

### PR DESCRIPTION
**Pull request by cuipinghuo**
moving test cases OCP-15328,OCP-16086,OCP-16087 from internal cucushift (https://github.com/openshift/cucushift/pull/7581) to verification-test for stage pipeline automation. 
@anpingli @emmajiafan @jianzhangbjz @chengzhang1016 Please have a review, thanks!

### CODE REVIEW

File: features/svc-catalog_asb/asb.feature [hunk 1]
```diff
@@ -142,6 +142,75 @@ Feature: Ansible-service-broker related scenarios
...
+    And I use the "openshift-ansible-service-broker" project
+    And I save the first service broker registry prefix to :prefix clipboard
+
+    # Swtich back to normal user and create first project
+    And I switch to the first user
+    Given I have a project
+
+    # Provision DB apb with dev plan
+    When I process and create:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/svc-catalog/serviceinstance-template.yaml |
```
Comment (liangxia) [inline features/svc-catalog_asb/asb.feature:159]: Why is the move reverting to deprecated v3-testfiles ?

File: features/svc-catalog_asb/asb.feature [hunk 2]
```diff
@@ -142,6 +142,75 @@ Feature: Ansible-service-broker related scenarios
...
+    Given I have a project
+
+    # Provision DB apb with dev plan
+    When I process and create:
+      | f | <%= ENV['BUSHSLICER_HOME'] %>/testdata/svc-catalog/serviceinstance-template.yaml                             |
+      | p | INSTANCE_NAME=<db_name>                                                                                      |
+      | p | CLASS_EXTERNAL_NAME=<db_name>                                                                                |
+      | p | PLAN_EXTERNAL_NAME=dev                                                                                       |
+      | p | SECRET_NAME=<db_secret_name>                                                                                 |
+      | p | INSTANCE_NAMESPACE=<%= project.name %>                                                                       |
```
Comment (liangxia) [inline features/svc-catalog_asb/asb.feature:164]: Extra white-spaces before the last `|`

Comment (cuipinghuo) [reply features/svc-catalog_asb/asb.feature:164]: > Extra white-spaces before the last `|`

sorry, I can't figure it out. What is it should like?

Comment (pruan-rht) [reply features/svc-catalog_asb/asb.feature:164]: @cuipinghuo it looks like the extra empty space ```    When I process and create:
      | f | <%= ENV['BUSHSLICER_HOME'] %>/testdata/svc-catalog/serviceinstance-template.yaml                             |```

Comment (cuipinghuo) [reply features/svc-catalog_asb/asb.feature]: @liangxia @pruan-rht thanks and finally know what is not right.

File: features/svc-catalog_asb/asb.feature [hunk 3]
```diff
@@ -142,6 +142,75 @@ Feature: Ansible-service-broker related scenarios
...
+    # Provision DB apb with dev plan
+    When I process and create:
+      | f | <%= ENV['BUSHSLICER_HOME'] %>/testdata/svc-catalog/serviceinstance-template.yaml |
+      | p | INSTANCE_NAME=<db_name>                                                          |
+      | p | CLASS_EXTERNAL_NAME=<db_name>                                                    |
+      | p | PLAN_EXTERNAL_NAME=dev                                                           |
+      | p | SECRET_NAME=<db_secret_name>                                                     |
+      | p | INSTANCE_NAMESPACE=<%= project.name %>                                           |
+    Then the step should succeed
+    And evaluation of `service_instance("<db_name>").uid(user: user)` is stored in the :uid1 clipboard
```
Comment (liangxia) [inline features/svc-catalog_asb/asb.feature:166]: No longer need `(user: user)` now.

**Pull request merged**